src/Utils/findSignalLabel.ts

```diff
@@ -0,0 +1,13 @@
+export interface PartnerOffer {
+  isAvailable: boolean | null | undefined
+}
+
+export interface CollectorSignals {
+  partnerOffer: PartnerOffer | null | undefined
+}
```

Comment:
If these types originate from Relay queries, we should utilize the generated type, vs type mirroring